vector-stores/spring-ai-weaviate-store/src/main/java/org/springframework/ai/vectorstore/weaviate/WeaviateVectorStoreOptions.java

```diff
@@ -48,4 +50,13 @@ public void setContentFieldName(String contentFieldName) {
 		this.contentFieldName = contentFieldName;
 	}
 
+	public String getMetaFieldPrefix() {
+		return metaFieldPrefix;
+	}
+
+	public void setMetaFieldPrefix(String metaFieldPrefix) {
+		Assert.notNull(metaFieldPrefix, "metaFieldPrefix can be empty but not null");
+		this.metaFieldPrefix = metaFieldPrefix;
+	}
```

Comment:
These methods are newly created, but since the class Javadoc already includes the `@since 1.1.0` tag, the @since tag was not added to the methods.